docs/filter-example/src/manifest.json

```
@@ -0,0 +1,25 @@
+{
+  "name": "My Visualizations",
+  "organization": "MyOrg",
```

Should this be "Data Studio DevRel"?
```suggestion
  "organization": "Data Studio DevRel",
```